mobile.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Trainwise - mobile page</title>
+
+</head>
+
+<body>
+
+    <h1>Trainwise</h1>
+
+
+
+
+
+
+
+</body>
+
+</html>
```

Comment:
What will go into this file? We talked about making the web app responsive (with media queries) thus no need for a separate mobile file.

Reply:
Meaning one html page for both the users that upload recordings and the managment dashboard?